addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -215,7 +215,7 @@
                         <field name="description" placeholder="Notes..."/>
                     </div>
                 </sheet>
-                <div class="o_attachment_preview"/>
+                <div class="o_attachment_preview center_attachment"/>
```

Comment:
I'd at least add prefix `o_`, just in case this conflicts with some other non-odoo classnames (e.g. from PDF.js lib)